src/components/02-structure/04-seal/seal.njk

```diff
@@ -1,7 +1,7 @@
 <div class="height-mobile bg-accent-cool-light padding-y-4">
 
   <div class="sds-seal">
-    <i class="sds fa-flag fa-2x margin-y-1 text-secondary"></i>
```

Comment:
original utility classes (margin-y-1 text-secondary) missing

Reply:
This are in the seal.config file